Modules/AuctionManager/AuctionManager.lua

```diff
@@ -338,9 +338,23 @@ function AuctionManager:HandleNotifyPass(data, sender)
 end
 
 function AuctionManager:ValidateBid(name, bid)
-    -- allow bid cancelling
-    if bid == nil then return true end
-    -- allow bid passing
+    -- bid cancelling
+    if bid == nil then
+        if CONSTANTS.AUCTION_TYPES_OPEN[self.auctionType] then
+            return false, CONSTANTS.AUCTION_COMM.DENY_BID_REASON.CANCELLING_NOT_ALLOWED
+        else
+            return true
+        end
+    end
+    -- bid passing
+    if bid == CONSTANTS.AUCTION_COMM.BID_PASS then
+        -- only allow passing if no bids have been placed in open auctions
+        if CONSTANTS.AUCTION_TYPES_OPEN[self.auctionType] and self.highestBid > 0 then
```

Comment:
This will disable passing for everyone, not only the person that already bid. Intention here would be to disable passing if we already bid (it's complementary to disabling canceling at all).